packages/utils/pack-up/tests/spawn.ts

```diff
@@ -100,7 +100,7 @@ const spawn = async (projectName: string): Promise<Project> => {
 
   return {
     cwd: tmpPath,
-    install: () => execute('yarn install --no-immutable'),
+    install: () => execute('yarn install'),
```

Comment:
would it be safer to only disable the inmutable install here?
```suggestion
    install: () => execute('YARN_ENABLE_IMMUTABLE_INSTALLS=false && yarn install'),
```

We should not allow immutable installs in other places imo.

Reply:
I tried that a while back, i've just implemented what should be the correct way to pass it to the command – if it doesn't work then imo we should use what i've got for now to keep moving.

Reply:
it didn't work...